README.md

```diff
@@ -29,22 +34,29 @@ length limit is based on a survey of the defaults for various JSON parsers.
 extension which MUST reference all the parameters.
 * Type: Data type for the parameter. In order to allow validation and limit the scope for implementation specific variations,
 parameters are typed.
-  * integer - may be signed or not as specified in the range parameter.
+  * integer - may be signed or not as specified in the range property.
   * string - aligned with JSON strings, these should be UTF-8 encoded and avoid control characters.  
   * enumerated - one of an ordered set of labels which MUST conform to the same limitations as parameter names.  
-  * boolean - may have the values false or true (lower case and unquoted as in JSON)
-  * For integer parameters the range specifies minimum and maximum values, separated by a comma, which MUST be integers themselves.
-  * For string parameters, the range specifies the maximum length of the string as an integer number of characters, not bytes. Again, based on a survey of parsers, try to keep strings shorter than 4095 characters.
+  * boolean - MUST have the value *false* or *true* (lower case and unquoted as in JSON)
+* Range: Further qualifies the valid values for a parameter depending on its type.
+  * For integer parameters, the range specifies minimum and maximum values, separated by a comma, which MUST be integers themselves.
+  * For string parameters, the range specifies the maximum length of the string as an integer number of characters, not bytes. Again, based on a survey of parsers, strings SHOULD be shorter than 4095 characters.
   * For enumerated parameters, the range is a comma separated ordered list of labels that are valid for the parameter. Enumerated parameters are case sensitive and MUST always be quoted, so they are JSON strings. 
-* Default: Default value for parameter, which MUST be consistent with the range limitations. If this is left blank then the parameter is mandatory 
+* Default: Default value for parameter, which MUST be consistent with the range limitations. If this is left blank then the parameter is mandatory (if parameters are defined)  
 
 ## Referencing Parameters
 
-Wherever a parameterised extension is referenced, include any parameters in an accompanying JSON file. If using an extensions directory, the JSON file MUST be named for the extension and included in the directory. For example, the example extension above would have an accompanying file *0000-example-extension.json* which might contain:
+Wherever a parameterised extension is referenced, any parameters MAY be included in an accompanying sidecar JSON file that
+uses the registered name with the filename extension `.json`. If using an 'extensions' directory, the JSON file MUST be 
+included at the top level of the directory and not a subdirectory. Another place that an extension may be referenced is
```

Comment:
> Another place that an extension may be referenced is `ocfl_layout.json` where the sidecar file should be alongside it in the Storage Root.

Is this the equivalent of saying?

> If `ocfl_layout.json` references an extension, then the extension's sidecar file must be in the Storage Root instead of the extensions directory.

Or should both locations be checked?

Reply:
Yes (to your first question).